packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/useLibrary.types.ts

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { GQLLibrary } from './useLibraries.types';
```

Comment:
Don't share types between different GraphQL queries, those two features are not coupled together

Reply:
Done